examples/ai-core/src/e2e/google-vertex.test.ts

```diff
@@ -26,6 +27,7 @@ const MODEL_VARIANTS = {
     // 'gemini-1.0-pro-001',
   ],
   embedding: ['textembedding-gecko', 'textembedding-gecko-multilingual'],
+  image: ['imagen-3.0-generate-001', 'imagen-3.0-fast-generate-001'],
```

Comment:
is it possible to run specific e2e tests easily? as in all image tests or a specific image test?

Reply:
Yes, you can use `-t <filter string>` so `-t image` to match `it` or `describe` strings. You can use `-t 'Chat Model'` or `-t 'Image Model'` for example to run all test cases under the respective `describe` subset. I added a note about this in the [README](https://github.com/vercel/ai/tree/main/examples/ai-core#end-to-end-provider-integration-tests) although I didn't fully elaborate as much as above.

Reply:
I don't know if you can use `-t` in some manner to specify you only want to run tests for a particular model, though. I have been commenting/uncommenting them in the list when I want to do that.